guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc

```diff
@@ -16,7 +16,7 @@ endif::[]
 This is required because the DHCP server in {RHEL} (ISC DHCP) does not provide an integration API for managing IPv6 records, therefore the {SmartProxy} DHCP plug-in that provides DHCP management is limited to IPv4 subnets.
 
 * You must deploy an external IPv4 HTTP proxy server.
-This is required because {Project} distributes content only over IPv4 networks, therefore you must use an IPv4 proxy to redirect that content to hosts in your IPv6 network.
+This is required because the Red Hat Content Delivery Network distributes content only over IPv4 networks, therefore you must use this proxy to pull content into the {Project} on your IPv6 network.
```

Comment:
I feel like this is Satellite-only; or, more precise, users that rely on content for RHEL from RH CDN.

Reply:
@maximiliankolb This section in the Foreman Installing Server Guide was referenced for the change: https://docs.theforeman.org/3.1/Installing_Server/index-satellite.html#requirements-for-installation-in-an-ipv6-network_satellite
Should I just condition it for Satellite instead?

Reply:
Yes, please create two versions, one for Satelite with RH CDN, and a generic one for all other content that might be IPv4 only.

Reply:
@maximiliankolb Let me know if I did this right. The second version is conditioned for Foreman and Katello but no mention of RH CDN. Just a CDN. Not sure if this is generic enough.